CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ See [GitHub releases](https://github.com/mll-lab/laravel-utils/releases).
 
 ## Unreleased
 
+## v6.0.0
+
+### Added
+
+- Support `mll-lab/php-utils` 2
+
+### Changed
+
+- Drop support for `mll-lab/php-utils` ^1.3
```

Comment:
Why? Seems like an unnecessary breaking change that causes churn.

Reply:
@kilya11 requested this change to keep the dependencies simple. 
I am actualy ok with both versions because both have pros and cons

Reply:
Sure, the simplicity has its cost. Let the [#1 contributor](https://github.com/mll-lab/laravel-utils/graphs/contributors) approve.